README.md

```diff
@@ -39,11 +39,19 @@ You should now receive an email whenever the changelog is updated.
 
 ---
 ### 18 July 2023
-The following changes are available in sandbox:
+The following changes are now available in sandbox:
```

Comment:
The update about the new error should be for both sandbox and production (if we are deploying to prod right now) and the gov-test-scenario that has been added for the error should also be mentioned as being available in sandbox.